kfdefs/base/monitoring/blackbox-exporter/probe.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: Probe
+metadata:
+  name: opf-prober
+spec:
+  jobName: opf-status
+  interval: 60s
+  module: http_2xx
+  prober:
+    url: blackbox-prober.opf-monitoring.svc.cluster.local:80
+    scheme: http
+    path: /probe
+  targets:
+    staticConfig:
+      static:
+      - https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/
+      - http://argo-server-opf-argo.apps.zero.massopen.cloud/
+      - http://superset-opf-superset.apps.zero.massopen.cloud/
+      - https://hue-opf-datacatalog.apps.zero.massopen.cloud/desktop/debug/is_alive
+      - http://thriftserver-opf-datacatalog.apps.zero.massopen.cloud/
+      - https://grafana-route-opf-monitoring.apps.zero.massopen.cloud/
+      - https://kibana-openshift-logging.apps.zero.massopen.cloud/api/status
+      - https://elasticsearch-openshift-logging.apps.zero.massopen.cloud/
```

Comment:
Are you able to authenticate to this endpoint without any certs?

Reply:
yeah seems like it requires certs, I wasn't completely sure how I would need to add this, will take a look :+1: